content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -23,6 +23,20 @@ in an old version of longhorn-instance-manager until it is detached (by scaling
 reattached (by scaling its consuming workload up). Consider scaling workloads down and back up again as soon as possible
 after upgrading from a version without this mechanism (v1.5.1 or older) to v{{< current-version >}}.
 
+### Default Priority Class
+
+Longhorn v1.6.0 introduces the default priority class `longhorn-critical` with the highest value to ensure longhorn pods will not be evicted when low system resources by the Kube scheduler.
+
+For upgrade:
+
+1. If all volumes are detached, the setting will be applied including user-managed components. The setting of the priority class will be updated because the source of truth of settings is the config map (longhorn-default-setting).
+2. If all volumes are detached and the setting of priority class is set to a priority class, the setting should be applied except for user-managed components. The setting of the priority class will not be updated.
+3. If volumes are attached, the setting should be applied except for user-managed components. The setting of the priority class will not be updated.
```

Comment:
```suggestion
- When all volumes are detached and the Priority Class default setting is not customized, the default Priority Class is applied to all Longhorn components. 

- When all volumes are detached and you have specified a value for the Priority Class default setting, the default Priority Class is applied only to system-managed components. The Priority Class default setting for user-deployed components is not updated.

- When one or more volumes are attached, the default Priority Class is applied only to system-managed components. The Priority Class default setting for user-deployed components is not updated.
```

Reply:
- This is not a procedure so we should use an unordered list.
- I rewrote the descriptions to clearly describe the conditions and affected components.

Reply:
Modified. Thanks

Reply:
And I modified the user-deployed components and system-managed components parts.
System-managed components would be affected only when the default Priority Class setting `priority-class` can be updated.